.github/workflows/ci-pipeline-arm64.yml

```diff
@@ -39,12 +39,15 @@ jobs:
       fail-fast: false
       matrix:
         cc: [gcc, clang]
-        os: ['ubuntu_16.04', 'ubuntu_18.04']
+        os: ['ubuntu_16.04', 'ubuntu_18.04', rocky_linux_8]
     steps:
       - uses: AutoModality/action-clean@v1.1.0
       - uses: actions/checkout@v2
       - run: sudo docker run -i -v `pwd`:/odp --privileged --shm-size 8g -e CC="${{matrix.cc}}"
                -e CONF="${CONF}" $CONTAINER_NAMESPACE/odp-ci-${{matrix.os}}-${ARCH}-native /odp/scripts/ci/build_${ARCH}.sh
+      - name: Failure log
```

Comment:
Not all jobs have this "Failure log" step, is this on purpose?

Reply:
Yep, I'm trying to figure out why configure is failing on arm64.

Reply:
OK. Point here was that shouldn't all jobs have the "Failure Log" step, why not have it?

Reply:
Added to all build tests.